switchmap/poller/snmp/snmp_manager.py

```
@@ -974,6 +1133,9 @@ def _update_cache(filename, group):
     Returns:
         None
     """
-    # Do update
-    with open(filename, "w+") as env:
-        env.write(group)
+    try:
+        with open(filename, "w+") as f_handle:
+            f_handle.write(group)
+    except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)